aten/src/ATen/core/Vitals.cpp

```diff
@@ -51,9 +75,19 @@ bool APIVitals::setVital(
     vital = &iter->second;
   }
 
-  vital->create(attr_name) << value;
+  if (force) {
+    vital->create(attr_name, /* force = */ true).force_write(value);
```

Comment:
here conditional also seems like unnecessary complication - if you had `write(value, force)` you could have avoided conditional